Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:alpine3.15 AS base
+FROM python:3.8.5-slim-buster AS base
 MAINTAINER Gengo Dev Team
 RUN apk add build-base linux-headers pcre-dev
```

Comment:
Is the base image `python:3.8.5-slim-buster` still alpine? If so, it's okay but I think usually debian based OS is used so don't we need to replace `apk` command to `apt`?

Reply:
The [base image](https://github.com/docker-library/python/blob/06e0db2fc2e32eb95771b23bc8ad36bf9450640a/3.8/slim-buster/Dockerfile) looks like Debian so `apt-get` might be needed